modules/api-custom-image-creating.adoc

```diff
@@ -77,8 +79,10 @@ The example YAML file includes the following information:
 <6> The version for the image. You can configure multiple versions for the same image. For this example, the version is 1.0.
 <7> An annotation that gives the user information about the Python packages and versions that are pre-installed in the image.
 <8> An annotation that specifies information such as the Python version, Jupyter version, or CUDA version.
-<9> An annotation that specifies whether the `ImageStream` version is the default version of the image. Set this field to `'true'` if the `ImageStream` version is the default. Otherwise, set it to `'false'`. You must specify the `opendatahub.io/workbench-image-recommended` annotation field if there are multiple versions of the image with different configurations. If you have only one version of the image, set the field to `'true'`.
-<10> The image registry URL to store the image stream. Store the image stream in an image registry outside of the cluster where {productname-short} is installed.
+<9> An annotation that specifies whether the `ImageStream` version is the default version of the image. Set this field to `'true'` if the `ImageStream` version is the default. Otherwise, set it to `'false'`. You must specify the `opendatahub.io/workbench-image-recommended` annotation field if there are multiple versions of the image with different configurations. If you have only one version of the image, set the field to `'true'`. 
+<10> An annotation that specifies whether the image version has tags that are outdated and out of the regular maintenance cycle.
```

Comment:
This is already poorly described in the original https://github.com/opendatahub-io/notebooks/edit/main/docs/workbench-imagestreams.md

Raising 

* https://github.com/opendatahub-io/notebooks/pull/1155

to fix it there.

Reply:
okay - for the purpose of this ticket, I will disregard :)

Reply:
What? Why? I thought you wanted a review on the new content?